tests/RestfulCreateEntityTestCase.test

```diff
@@ -178,31 +178,73 @@ class RestfulCreateEntityTestCase extends RestfulCurlBaseTestCase {
 
   }
 
+  public function testFileUploadOptions() {
+    variable_set('restful_file_upload', TRUE);
+    variable_set('restful_file_upload_allow_anonymous_user', TRUE);
+
+    // Upload a permanent file.
+    $return = $this->uploadFile();
+    $this->assertEqual($return['code'], 200, 'File uploaded.');
+
+    $return_array = drupal_json_decode($return['body']);
+    $file = file_load($return_array['data'][0]['id']);
+
+    // Make sure it's the same file that was uploaded.
+    $this->compareFile($file);
+    // File should be permanent.
+    $this->assertEqual($file->status, FILE_STATUS_PERMANENT, 'File status is permanent.');
+    $usage = file_usage_list($file);
+    $this->assertEqual(reset($usage['restful']['files']), 1, 'File usage is registered by RESTful.');
+
+
+    // Upload a temporary file
+    //drupal_static_reset('restful_get_restful_handler');
+    cache_clear_all();
+print_r(array_keys(restful_get_restful_plugins()));
+    $handler = restful_get_restful_handler('files', 1, 0);
```

Comment:
This is returning NULL but it's not obvious to me why. Locally I'm seeing `files` but Travis is not:
```
Array
(
    [0] => per_role_content__1_0:anonymous user
    [1] => per_role_content__1_0:authenticated user
    [2] => per_role_content__1_0:administrator
    [3] => csrf_token
    [4] => discovery__1_0
    [5] => login_cookie__1_0
    [6] => users__1_0
    [7] => node_user__1_0
    [8] => watchdog
    [9] => articles__1_0
    [10] => articles__1_1
    [11] => articles__1_2
    [12] => articles__1_4
    [13] => articles__1_5
    [14] => articles__1_6
    [15] => articles__1_7
    [16] => articles__2_0
    [17] => tags__1_0
    [18] => variable__1_0
    [19] => db_query_test__1_0
    [20] => entity_tests__1_0
    [21] => main__1_0
    [22] => main__1_1
    [23] => main__1_2
    [24] => main__1_3
    [25] => main__1_4
    [26] => main__1_5
    [27] => tests__1_0
    [28] => test_articles__1_0
    [29] => test_articles__1_1
    [30] => test_articles__1_2
    [31] => test_articles__1_3
    [32] => test_articles__1_4
    [33] => restful_test_translatable_entity__1_0
    [34] => test_tags__1_0
)
```